README.md

```diff
@@ -104,43 +102,33 @@ for _, v := range list {
 }
 ```
 
-For full information on every method, see the [Godoc]()
+For full information on every method, see the [Godoc]().
```

Comment:
Why is this link empty?

Reply:
Found the link, we should sanity check that there is automation that keeps these up to date.

Reply:
GoDoc updates everyday and pulls the documentation from the comments for each function.

https://godoc.org/-/about